python/3.10/build.yaml

```
@@ -10,9 +10,9 @@ codenotary:
   signer: notary@home-assistant.io
   base_image: notary@home-assistant.io
 args:
-  PYTHON_VERSION: 3.7.12
+  PYTHON_VERSION: 3.10.4
   PIP_VERSION: 22.0.3
-  GPG_KEY: 0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D
+  GPG_KEY: E3FF2839C048B25C084DEBE9B26995E310250568
```

For Python 3.10 we need to use this key

```suggestion
  GPG_KEY: A035C8C19219BA821ECEA86B64E628F8D684696D
```

Looking back at my previous comment, this was the key that Python 3.9 was using (I guess the 3.9/3.10 where swapped in this PR by accident).